server/datastore/mysql/software_test.go

```diff
@@ -513,7 +521,10 @@ func testSoftwareList(t *testing.T, ds *Datastore) {
 		require.NoError(t, err)
 
 		require.Len(t, software, 1)
-		expected := []fleet.Software{foo003}
+
+		foo003WithCount := foo003
+		foo003WithCount.HostCount = 1
+		expected := []fleet.Software{foo003WithCount}
 		test.ElementsMatchSkipID(t, software, expected)
```

Comment:
Nit: Use `ElementsMatchSkipIDAndHostCount`? Same in other places where you artificially set the `HostCount`.

Reply:
I did that, but then I realized that I wanted to make sure the counts were correct here as a test. So I updated them to match the scope of the filter. So in this case, foo003 is in one host within that team, while it's in 2 in the global scope.